src/Compiler.Dynamic/Compiler.Dynamic.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\WebForms\WebForms.csproj" />
+    <ProjectReference Include="..\Extensions\Extensions.csproj" />
```

Comment:
let's remove tthis and connect it up at some other layer

Reply:
I am thinking creating an extension method like below and calling from program.cs... what you think ? Or should we create this project as a dependency in the Webform project and add as a part of ```AddWebForms()``` extension method ? LMK what you think @twsouthwick ?
``` C#
    public static void AddWebFormsExtensionNameSpae(this IServiceCollection services)
    {
        services.AddOptions<PageCompilationOptions>()
            .Configure(options =>
            {
                options.AddTypeNamespace<ScriptManager>("asp");
                options.AddTypeNamespace<ListView>("asp");
            });
    }
````

Reply:
This looks good - let's put it in the extensions project but maybe off of the system web adapters builder:

```csharp
public static ISystemWebAdapterBuilder AddWebFormsExtensions(this ISystemWebAdapterBuilder builder)
{
...
}
```